test/test_model_structure_variabletimestep.jl

```diff
@@ -46,13 +46,7 @@ last_A = 2150
 
 m = Model()
 set_dimension!(m, :time, years)
-
-# ArgumentError in v1.0-v1.2, ErrorException in v1.3
-if(VERSION < v"1.3.0")
-    @test_throws ArgumentError add_comp!(m, A, after=:B)
-else
-    @test_throws ErrorException add_comp!(m, A, after=:B)
-end
+@test_throws ErrorException add_comp!(m, A, after=:B)
```

Comment:
this will now fail v1.2 and v1.3 tests, could move to a separate PR if that's an issue